pkg/reconciler/openshift/tektonpipeline/extension.go

```diff
@@ -75,9 +83,11 @@ type openshiftExtension struct {
 
 func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Transformer {
 	return []mf.Transformer{
+		common.InjectLabelOnNamespace(monitoringLabel),
```

Comment:
This might need to change with the 1.5 change on disabling the service monitor, right ?

Reply:
yep after this pr I will handle that in openshift extension